src_assets/common/assets/web/config.html

```diff
@@ -1142,6 +1242,12 @@ <h2 class="accordion-header">
               "install_steam_audio_drivers": "enabled",
               "adapter_name": "",
               "output_name": "",
+              "display_device_prep": "no_operation",
+              "resolution_change": "no_operation",
+              "manual_resolution": "",
+              "refresh_rate_change": "no_operation",
```

Comment:
I think we should default `resolution_change` and `refresh_rate_change` to `automatic` too, but let's only change the resolution if the `sops` parameter passed to launch/resume is 1. That value is tied to the "Optimize game settings" option on the client-side.